src/rpc/server/basic.zig

```diff
@@ -65,7 +97,7 @@ pub fn acceptAndServeConnection(server_ctx: *server.Context) !void {
         .HEAD, .GET => switch (requests.getRequestTargetResolve(
```

Comment:
Maybe one function can handle HEAD and GET, and another function can handle POST, so this entire switch statement would just be two function calls.

Reply:
ea44b992dc2524ce6ff0fa4541a52199ca970ceb